src/main/play-doc/operation/Licensing.md

```diff
@@ -0,0 +1,107 @@
+`commercial.credentials` it’s important to use the email associated with your organization's license, and not a personal
+account.  If enterprise environments this is usually your corporate email.
+2. Download your `license` file manually.  You can find your license by visiting 
+https://www.lightbend.com/product/conductr/license
+3. Copy & paste the contents of this page into your `~/.lightbend/license` file.
+4. Execute `conduct load-license --offline` at the command line.
+
+```
+$ conduct load-license --offline
+Skipping downloading license from Lightbend.com
+Loading license into ConductR at 192.168.10.1
+
+Licensed To: aaaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaaa
+Expires In: 365 days (Fri 27 Jul 2018 16:09PM)
+Grants: akka-sbr, cinnamon, conductr, fortify
+
+License successfully loaded
+```
+
+### Viewing the current license ConductR cluster
+
+When you issue `conduct info` you will see the same license details that were returned when the license was first
+loaded.
+
+### Updating your License
+
+If you need to update the license on an already licensed cluster you can do so using the `--force` command switch. This
+is useful when you want to change the license to a different user or when your license is close to expiring.  Enter
+`conduct load-license --offline` and the command line and then follow the prompts to load a new license.
+
+## Automate Licensing at Startup
+
+When a license is loaded it is submitted to the ConductR cluster and distributed to all the ConductR core nodes.  The
+license resides in memory, so in the event of a complete cluster shutdown the data is lost.  In production this would
+be an unlikely scenario as your cluster should be able to tolerate the failure of some nodes.
+
+In non-production environments it may be commonplace to shutdown whole clusters for a number of reasons: cluster is no
+longer required, to save on public cloud instance charges, etc.
+
+In any environment, whenever a cluster starts up it is important to license before bundles are loaded and run again.
+By retrieving your license manually as described in "Offline Mode", you can automate the licensing of a cluster whenever
+it's first started up.
+
+Remember that licenses expire, so set a reminder to occassionally retrieve an updated `./.conductr/license` file for 
+your deployment script.
+
+## Other ConductR Licensing Information and Caveats
+
+- Anyone using the CLI to work with a licensed cluster will be using the same license.
+- Bundles, like licenses, are lost in the event of a total cluster shutdown.  You should plan for this contingency 
+when starting your cluster in the same manner you plan to license it.  You should load all required support bundles
+(`visualizer`, `haproxy`, `continuous-delivery`, etc.) as well as your latest running app bundles.
+- While there is at least one node of the cluster up, the license will be intact.
+- Future versions of ConductR will persist the license to disk in the cluster.
```

Comment:
Let's note the backup and restore feature, https://github.com/typesafehub/conductr-cli/pull/517, instead. For the immediate future, users preparing for full data center shutdown recovery should have their license and bundle backups in the plan. When/If CRDT state is persisted, the two are a shared concern. Or, a licensed but empty post-disaster cluster isn't very useful.